src/toniarts/openkeeper/game/trigger/TriggerActionData.java

```diff
@@ -28,7 +28,6 @@
 public class TriggerActionData extends TriggerData {
 
     private TriggerAction.ActionType actionType;
-    private static final Logger logger = Logger.getLogger(TriggerActionData.class.getName());
```

Comment:
I think this is not good idea to remove comments. Some of them TODO comments and necessary in future